indico/modules/events/abstracts/templates/_abstract/review.html

```diff
@@ -0,0 +1,85 @@
+{% from 'events/abstracts/_abstract/_common.html' import render_edited_hint %}
+
+{% macro render_abstract_reviews(abstract) %}
+    <div id="abstract-reviews" class="i-timeline">
+        {% set reviews = abstract.reviews|selectattr('track', 'equalto', none)|list %}
+        {% if reviews %}
+            <h3>{% trans %}No track{% endtrans %}</h3>
+            {% for review in reviews %}
+                {{ _render_review(review) }}
+            {% endfor %}
+        {% endif %}
+        {% for track in abstract.reviewed_for_tracks %}
+            {% set reviews = abstract.reviews|selectattr('track', 'equalto', track)|list %}
+            {% if reviews %}
+                <h3>
+                    {% trans title=track.title -%}
+                        Track: {{ title }}
+                    {%- endtrans %}
+                </h3>
+                {% for review in reviews %}
+                    {{ _render_review(review) }}
+                {% endfor %}
+            {% endif %}
+        {% endfor %}
+    </div>
+{% endmacro %}
+
+{% macro _render_review_state(review) %}
+    {% set mapping = {'accept': 'success',
+                      'reject': 'error',
+                      'merge': 'visited',
+                      'mark_as_dupe': 'strong'} %}
+    <span class="i-tag outline {{ mapping[review.proposed_action.name] }}">
+        {{ review.proposed_action.title -}}
+    </span>
+{% endmacro %}
+
+{% macro _render_review(review) %}
+    <div class="i-timeline-item">
+        <div class="i-timeline-item-label action abstract-comment-pic"></div>
+        <div class="i-timeline-item-box header-indicator">
+            <div class="i-box-header flex">
+                <div class="stretch">
+                    {% trans name=review.user.name, date=review.created_dt|format_human_date -%}
+                        <strong>{{ name }}</strong> reviewed on {{ date }}
```

Comment:
In this case I guess that removing the preposition would sound better. "Reviewed 21/10/2016" doesn't look that bad IMHO.